perceval/backends/git.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, uri, gitpath, cache=None, origin=None):
         self.gitpath = gitpath
 
     @metadata
-    def fetch(self, from_date=DEFAULT_DATETIME):
+    def fetch(self, from_date=DEFAULT_DATETIME, branches=[]):
```

Comment:
`branches` cannot has an empty list nor a list as default value. It is a bad practice in Python and can hide bugs.
Please read [Common Gotchas in Python](http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments) to know more about it.